### DISCUSSION TITLE
Most Stable Saleor Docker Compose way to Deploy

### DISCUSSION THREAD

**Discussion by inf3cti0n95** [saleor/saleor#12374] [Deployment]
It is very confusing to understand which Saleor Core works with which Dashboard and which Frontend (react?, nextjs? which fork)

can someone please clarify, I need to migrate a website immediately to saleor.

**Comment by timuric** [comment]
Hey @inf3cti0n95 please check https://github.com/saleor/saleor-platform
Alternatively consider Saleor Cloud which provides worry free upgrades and devops

**Comment by karolkielecki** [comment]
Hi @inf3cti0n95, If your goal is production, please remember that the `saleor-platform` repo has been designed only for local development, as stated in the readme section. The easiest way to deploy your project to production is to use mentioned above [Saleor Cloud](https://cloud.saleor.io/signup). You can create a free developer account to test it. 

Re core and dashboard versions: we recommend using the latest stable releases, which currently are v3.12+.

Re frontend:
First, Saleor is headless, meaning you can connect any fronted you want. [saleor-next-starter](https://github.com/saleor/saleor-next-starter) is a Next.js front-end boilerplate for quickly getting Saleor up and running. The scope of the starter is limited by design and focuses on solving the core challenges of integrating with Saleor – leaving the rest up to you.

It contains:

- GraphQL setup (with codegen)
- Authentication
- Server-side calls

The difference between [saleor-next-starter](https://github.com/saleor/saleor-next-starter) and [react-storefront](https://github.com/saleor/react-storefront) is that the latter is a complete demo repository you can use to explore the features of Saleor. Starter allows you to start building your own Saleor storefront.

**Comment by patrys** [comment]
We use matching major+minor versions for compatible releases. For example, the latest dashboard version `3.12.x` will always work with the latest core version `3.12.x`.
